lib/jekyll-github-metadata/repository_finder.rb

```diff
@@ -45,16 +45,14 @@ def nwo_from_config
         repo if repo && repo.is_a?(String) && repo.include?("/")
       end
 
-      def git_exe_path
-        ENV["PATH"].to_s
-          .split(File::PATH_SEPARATOR)
-          .map { |path| File.join(path, "git") }
-          .find { |path| File.exist?(path) }
-      end
-
       def git_remotes
-        return [] if git_exe_path.nil?
-        `#{git_exe_path} remote --verbose`.to_s.strip.split("\n")
+        output = ""
+        begin
+          _process, output = Jekyll::Utils::Exec.run("git", "remote", "--verbose")
+        rescue Errno::ENOENT => e
+          Jekyll.logger.warn "Not Installed:", e.message
+        end
+        output.to_s.strip.split("\n")
```

Comment:
Could this be simplified to...?

```ruby
def git_remotes
  _process, output = Jekyll::Utils::Exec.run("git", "remote", "--verbose")
  output.to_s.strip.split("\n")
rescue Errno::ENOENT => e
  Jekyll.logger.warn "Not Installed:", e.message
end
```

Reply:
@ashmaroli Are you willing to update this PR to use @benbalter's suggestion?

Reply:
> ~~@ashmaroli Are you willing to update this PR to use @benbalter's suggestion?~~

Oops! This is meant for @GICodeWarrior, not @ashmaroli  😄